guides/doc-Upgrading_and_Updating/topics/upgrading_process_overview.adoc

```diff
@@ -73,26 +76,29 @@ You can upgrade to {ProjectName} {ProjectVersion} from {ProjectName} {ProjectVer
 For more information, see the {Project} {ProjectVersionPrevious} Upgrade documentation.
 endif::[]
 
-.Overview of {Project} {ProjectVersion} Upgrade Paths
-ifdef::satellite[]
-image::satellite_6.4_upgrade_paths.png[Overview of {Project} {ProjectVersion} Upgrade Paths]
-
-WARNING: Upgrading from the Beta to GA version is not supported.
-endif::[]
-
-The high level steps in upgrading to {Project} {ProjectVersion} are as follows.
+The high-level steps in upgrading {Project} to {ProjectVersion} are as follows.
 
 ifdef::satellite[]
 . Clone your existing {ProjectServer}s. For more information, see xref:cloning_satellite_server[].
 endif::[]
 . Upgrade {ProjectServer} to {ProjectVersion}.
++
+ifdef::satellite[]
+During an upgrade of {ProjectServer}, you must observe the correct upgrade path depending on your network environment:
++
+.Overview of {ProjectServer} Upgrade Paths in Connected and Disconnected Network Environments
+image::satellite_6.4_upgrade_paths.png[Overview of {ProjectServer} Upgrade Paths in Connected and Disconnected Network Environments]
```

Comment:
having this picture in the middle of the "high level steps" seems weird to me. it's a detail that doesn't *really* influence the high level steps (those are, in both cases, "upgrade satellite first", "then upgrade capsules")

Reply:
So where would you like it better?

Reply:
If I would be writing this doc fresh (I am not), I wouldn't have included this picture in this paragraph at all.

But it was included and I don't have enough background knowledge to know why exactly, so I would probably just not move it (so leave it above the "The high level steps in upgrading" line and the "no upgrades from beta" warning)

Reply:
I'll comment it out and we can get back to it later.